<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkListDialog</class>
 <widget class="QDialog" name="BookmarkListDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>451</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Bookmarks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>18</number>
     </attribute>
     <column>
      <property name="text">
       <string>Position</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Text</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionRemoveBookmark">
   <property name="text">
    <string>Remove Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionRemoveAllBookmarks">
   <property name="text">
    <string>Remove all bookmarks</string>
   </property>
   <property name="toolTip">
    <string>Remove all bookmarks</string>
   </property>
   <property name="shortcut">
    <string>Alt+Backspace</string>
   </property>
  </action>
  <action name="actionSelectBookmark">
   <property name="text">
    <string>SelectBookmark</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
